test/unit/api_spec.js

```diff
@@ -80,7 +80,7 @@ describe('api', function() {
         loadingTask.promise
       ];
       Promise.all(promises).then(function (data) {
-        expect((data[0].loaded / data[0].total) > 0).toEqual(true);
+        expect((data[0].loaded / data[0].total) >= 0).toEqual(true);
```

Comment:
Would the following work too? Doing it like this would assert that the file is loaded (i.e., the `total` attribute is nonzero) while also asserting and explaining why the `loaded` attribute is zero.

```javascript
// The `loaded` attribute represents the number of fetched full chunks. For stream
// loading, the received data is less than the size of a full chunk, so it is always zero.
expect(data[0].loaded).toEqual(0);
expect(data[0].total >= 0).toEqual(true);
```

Reply:
It's *not* expected that `loaded` is ever zero normally, it's probably just a timing related artifact on e.g. newer Node.js versions.

Reply:
Thank you. I guess I'm just overthinking this.